filesystem/replicator_test.go

```diff
@@ -136,4 +136,45 @@ import (
 		}
 	 })
  }
+ func TestReplicateDeletionCallBack(t *testing.T) {
+		t.Run("test for destination folder gets created", func(t *testing.T) {
```

Comment:
We can remove this 1st test and just keep the 2nd test that tests for both creation and permission.